tests/src/Functional/CorporateFooterRenderTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_corporate_blocks\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Test footer block rendering.
+ */
+class CorporateFooterRenderTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'block',
+    'oe_corporate_blocks',
+    'oe_corporate_site_info',
+    'system',
+    'user',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    \Drupal::configFactory()
+      ->getEditable('oe_corporate_site_info.settings')
+      ->set('accessibility', 'https://example.com/accessibility')
+      ->save();
```

Comment:
We should do this change inside the test, after checking that no accessibility link is shown in the default value (no accessibility page set).